API.md

```diff
@@ -0,0 +1,66 @@
+# AmperkaCCS811 API
+
+## `class AmperkaCCS811`
+
+Create an object of type `CCS811` to communicate with a particular [Air Quality Sensor](https://amperka.ru/product/sensor-co2-ccs811-with-case).
+
+### `AmperkaCCS811(uint8_t i2cAddress = 0x5A)`
+
+Constructs a new object. The argument `i2cAddress` specifies the board I²C address in HEX value, which is `0x5A` factory-default but can be changed to `0x5B`. If omitted, the board I²C address is `0x5A`.
+
+### `bool begin(TwoWire* wire = &Wire)`
+
+Initializes the given interface, prepares the board for communication. Returns `true` device is set up, otherwise `false`. The argument `wire` is a pointer to the TwoWire interface. If omitted, the connection starts on default hardware I²C.
```

Comment:
```suggestion
Initializes the given interface, prepares the board for communication. Returns `true` device is set up, otherwise `false`. The argument `wire` is a pointer to the TwoWire interface. If omitted, the connection starts with default hardware I²C.
```